bundles/AdminBundle/Security/Event/LogoutListener.php

```diff
@@ -114,7 +114,7 @@ public function onLogoutSuccess(Request $request): RedirectResponse|Response
         }
 
         // cleanup pimcore-cookies => 315554400 => strtotime('1980-01-01')
-        $response->headers->setCookie(new Cookie('pimcore_opentabs', null, 315554400, '/'));
+        $response->headers->setCookie(new Cookie('pimcore_opentabs', null, 315554400));
         $response->headers->clearCookie('pimcore_admin_sid', '/', null, false, true);
```

Comment:
I'm not sure but does the auto-secure cookies get cleared when `$secure` parameter is passed `false`?
As far as I remember, it needs to be null in order to set an auto-secure cookie.

Reply:
@aashan10 I'd remove it here as well and let the default value, as we do using `setCookie()`

Reply:
Oh no ... tricky one ... `clearCookie()` has `$secure = false` as the default, so we need to set it to `null` and cannot rely on the defaults.

Reply:
Fixed, good catch, thanks! 👍